tests/conn_can/Makefile

```diff
@@ -8,7 +8,15 @@ BOARD_INSUFFICIENT_MEMORY := arduino-duemilanove arduino-mega2560 arduino-uno \
                              nucleo-l053r8 stm32f0discovery telosb \
                              waspmote-pro wsn430-v1_3b wsn430-v1_4 z1
 
-CFLAGS += -DLOG_LEVEL=LOG_ALL
+BOARDS_WITH_STM32_CANDEV := nucleo-f072rb \
```

Comment:
what's that for? actually it would be better to have an `ifeq` or something matching these boards and then add `USEMODULE = periph_can` bc otherwise CAN is not pulled in. At least I had to specify the USEMODULE explicitly for my nucleo-f103rb, otherwise there was no can device available

Reply:
This is a left-over, I'll remove.

The boards supporting CAN should have a `FEATURES_PROVIDED += periph_can` in their `Makefile.features` to pull it in (nucleo-f413zh has it, we can expand to other boards too)